app/controllers/logs_controller.rb

```diff
@@ -95,24 +95,24 @@ def stats_service
     case params[:what]
     when 'poundage'
       if params[:region_id].nil?
-        t = LogPart.sum(:weight) + Region.where('prior_lbs_rescued IS NOT NULL').sum('prior_lbs_rescued')
+        total = LogPart.sum(:weight) + Region.where('prior_lbs_rescued IS NOT NULL').sum('prior_lbs_rescued')
       else
-        r = params[:region_id]
-        @region = Region.find(r)
-        t = Log.joins(:log_parts).where('region_id = ? AND complete', r).sum('weight').to_f
-        t += @region.prior_lbs_rescued.to_f unless @region.nil? or @region.prior_lbs_rescued.nil?
+        region_id = params[:region_id]
+        @region = Region.find(region_id)
+        total = Log.joins(:log_parts).where('region_id = ? AND complete', region_id).sum('weight').to_f
+        total += @region.prior_lbs_rescued.to_f unless @region.nil? or @region.prior_lbs_rescued.nil?
       end
-      render :text => t.to_s
+      render :text => total.to_s
     when 'wordcloud'
       words = {}
-      LogPart.select('description').where('description IS NOT NULL').each{ |l|
-        l.description.strip.split(/\s*\,\s*/).each{ |w|
-          w = w.strip.downcase.tr(',', '')
-          next if w =~ /(nothing|no |none)/ or w =~ /etc/ or w =~ /n\/a/ or w =~ /misc/
+      LogPart.select('description').where('description IS NOT NULL').each{ |log_part|
+        log_part.description.strip.split(/\s*\,\s*/).each{ |word|
+          word = word.strip.downcase.tr(',', '')
+          next if word =~ /(nothing|no |none)/ or word =~ /etc/ or word =~ /n\/a/ or word =~ /misc/
           # people cannot seem to spell the most delicious fruit correctly
-          w = 'avocados' if w == 'avacados' or w == 'avocadoes' or w == 'avocado'
-          words[w] = 0 if words[w].nil?
-          words[w] += 1
+          word = 'avocados' if word == 'avacados' or word == 'avocadoes' or word == 'avocado'
```

Comment:
could clean this up a little by: `%w(avacados avocadoes avocado).include?(word)`